### ISSUE TITLE
admin user with password can't create bucket(permission error?)

### ISSUE THREAD

**Issue by echoface** [minio/minio#15441]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

OS: 
Linux VM-16-15-ubuntu 5.4.0-77-generic #86-Ubuntu SMP Thu Jun 17 02:35:03 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

minio Version: 
./minio -version
minio version RELEASE.2022-07-29T19-40-48Z (commit-id=067ebab9d8b821a900bbbc7823b36c95ef96007c)
Runtime: go1.18.4 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2022 MinIO, Inc.

```
./minio server data --console-address ":9001"
MinIO Object Storage Server
Copyright: 2015-2022 MinIO, Inc.
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Version: RELEASE.2022-07-29T19-40-48Z (go1.18.4 linux/amd64)

Status:         1 Online, 0 Offline.
API: http://10.0.16.15:9000  http://127.0.0.1:9000
RootUser: minioadmin
RootPass: minioadmin
Console: http://10.0.16.15:9001 http://127.0.0.1:9001
RootUser: minioadmin
RootPass: minioadmin

Command-line: https://docs.min.io/docs/minio-client-quickstart-guide
   $ mc alias set myminio http://10.0.16.15:9000 minioadmin minioadmin

Documentation: https://docs.min.io

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ You are running an older version of MinIO released 9 hours ago ┃
┃ Update: Run `mc admin update`                                  ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
```
## Expected Behavior

can create an manage bucket after login with minioadmin:minioadmin

## Current Behavior

<img width="1574" alt="image" src="https://user-images.githubusercontent.com/10153981/181920614-5283cb66-81c0-4995-a08c-b12feab4c2ff.png">

## Steps to Reproduce (for bugs)

1. download latest version of minio
2. run it with `/minio server data --console-address ":9001"`
3. config a nginx proxy
```
server {
    listen 80;
    server_name os.xxxxxxxxxx.com;
    access_log      /var/log/nginx/minio.com_access.log ;
    error_log       /var/log/nginx/minio.com_error.log ;
    location / {
        proxy_pass       http://localhost:9001;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-Proto $scheme;
    }
}

server {
    listen 80;
    server_name api.xxxxxxxx.com;
    access_log      /var/log/nginx/minio.com_access.log ;
    error_log       /var/log/nginx/minio.com_error.log ;
    location / {
        proxy_pass       http://localhost:9000;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-Proto $scheme;
    }
}
```
5. login by root user and defaults psw: minioadmin:minioadmin

## Your Environment
* Operating System and version (`uname -a`):
 Linux VM-16-15-ubuntu 5.4.0-77-generic #86-Ubuntu SMP Thu Jun 17 02:35:03 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

**Comment by harshavardhana** [comment]
Please upgrade to the latest release @echoface - fixed in #15437

**Comment by echoface** [comment]
thanks so much.
